inference-engine/tests/functional/plugin/shared/src/behavior/set_blob.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/set_blob.hpp"
+#include <single_layer_tests/cum_sum.hpp>
```

Comment:
Why? `#include <single_layer_tests/cum_sum.hpp>`

Reply:
I used CumSum layer to build ngraph fucntion because this layer supports 1D - 5D input tensor

Reply:
I just used CumSum layer to build ngraph fucntion

Reply:
As I can see, you use all defined methods from `CumSum` and overload some methods from 'LayerTestsCommon'. Maybe you should inherit directly from 'CumSumLayerTests'